clang/lib/CIR/CodeGen/CIRGenBuiltinAArch64.cpp

```diff
@@ -2818,7 +2818,7 @@ static mlir::Value emitCommonNeonSISDBuiltinExpr(
   case NEON::BI__builtin_neon_vqaddh_u16:
     llvm_unreachable(" neon_vqaddh_u16 NYI ");
   case NEON::BI__builtin_neon_vqadds_s32:
-    llvm_unreachable(" neon_vqadds_s32 NYI ");
+    return builder.createAdd(ops[0], ops[1], true, true, true);
```

Comment:
Maybe go the extra length and also handle the sub version while here? (assuming there's a sub version of the neon intrinsic)

Reply:
sure, will do it